packages/client/utils/draftjs/convertHtmlToTaskContent.ts

```diff
@@ -0,0 +1,16 @@
+import {ContentState, convertFromHTML} from 'draft-js'
+import {convertStateToRaw, removeSpaces} from './convertToTaskContent'
+import simpleDOMBuilder from './simpleDOMBuilder'
+
+export const convertHtmlToTaskContent = (spacedHtml: string) => {
```

Comment:
+1 It would be safe to move this to server side and always use `simpleDOMBuilder`. Mid-term we will move away from draft-js anyways.

Reply:
cool, I put them in the same `draftjs` folder to make it easier to remove them all at once later